src/components/organisms/FontSection.tsx

```diff
@@ -20,38 +22,73 @@ const FontSection = () => {
   const [scaleFactor, setScaleFactor] = useState(designData.font.scaleFactor);
   const { headingFontName, parragraphFontName } = designData.font;
 
+  const [errorMsg, setErrorMsg] = useState('');
+
+  function toTitleCase(str) {
+      return str.toLowerCase().replace(/(^|\s)\S/g, function(t) { return t.toUpperCase() });
+    }
+
   useEffect(() => {
-    setHeading(headingFontName);
-    setParagraph(parragraphFontName);
+    setHeading(toTitleCase(headingFontName));
+    setParagraph(toTitleCase(parragraphFontName));
   }, []);
 
   useEffect(() => {
-    const newHeading = headingFontName.replace(/\s/g, '+');
-    const newParagraph = parragraphFontName.replace(/\s/g, '+');
+    const newHeading = toTitleCase(headingFontName).replace(/\s/g, '+');
+    const newParagraph = toTitleCase(parragraphFontName).replace(/\s/g, '+');
     const headingLink = `${GOOGLE_FONTS_URL}${newHeading}${WEIGHT_QUERY}`;
     const paragraphLink = `${GOOGLE_FONTS_URL}${newParagraph}${WEIGHT_QUERY}`;
-    const link1 = document.createElement('link');
-    const link2 = document.createElement('link');
 
-    const links = document.getElementsByTagName('link');
+    const fetchFont = async (url, type) => {
+      let response;
+      try {
+        response = await axios.get(url);
+        return response.status === 200;
+      } catch (error) {
+        if (type === 1) setErrorMsg(`Font ${headingFontName} not found.`);
+        if (type === 2) setErrorMsg(`Font ${parragraphFontName} not found.`);
```

Comment:
Change the error message to Spanish

Reply:
the site is in english, does not require spanish alerts